SalesforceFoundation/HEDAP
# Refactoring custom label references in HEDA setting page

# Critical Changes

# Changes

- We've changed the custom label references on the HEDA Settings page to access custom labels directly. Specifically, we've replaced all $Label.hed.(LabelName) with $Label.c.(LabelName). This has the added benefit of cleaning up our code base and ensuring consistency and maintainability.

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
1.  Set up a new dev org#1
2. Create a dummy package
3. Add all heda components to the dummy package
4. Set up another new dev org#2
5. Install the dummy package in dev org#2 
6. Make sure everything in HEDA setting page still working as before
